src/TaskEither.ts

```diff
@@ -174,17 +174,21 @@ export const tryCatch = <L, A>(f: Lazy<Promise<A>>, onrejected: (reason: {}) =>
 }
 
 /** Convert a node style callback function to one returning a `TaskEither` */
-export function taskify<L, R>(f: (cb: (e: L, r: R) => void) => void): () => TaskEither<L, R>
-export function taskify<A, L, R>(f: (a: A, cb: (e: L, r: R) => void) => void): (a: A) => TaskEither<L, R>
-export function taskify<A, B, L, R>(f: (a: A, b: B, cb: (e: L, r: R) => void) => void): (a: A, b: B) => TaskEither<L, R>
+export function taskify<L, R>(f: (cb: (e: L | null | undefined, r: R) => void) => void): () => TaskEither<L, R>
```

Comment:
`r` is still required, I think we should change it to `r?: R`

Reply:
Don't you think we should close this and open a new PR to fix that?